config/application.rb

```diff
@@ -179,6 +179,12 @@ class Application < Rails::Application
         end
         child.helper helpers
       end
+
+      Plugin.all.each do |plugin|
+        plugin.to_prepare_callbacks.each do |callback|
```

Comment:
There migth be an issue, when developer renames the helper, but I guess in that case one should really restart the service, so it's good to fail here so that he/she notices